FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +20,26 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard let self = self else { return }
```

Comment:
Following from my previous comments, if we don't need a reference to the current `instance`, there's no need to upgrade it from `weak`. We can just check for `nil` instead:
```suggestion
			guard self != nil else { return }
```

Reply:
Now self is needed for the `mapRootToFeedImages`. Could extract this method to a class of its own, but this feels like a bit to much at this point.

Reply:
But the `mapRootToFeedImages` does not need to be an `instance` method, it could be marked as `static`. Also, it does not even need to be declared in `RemoteFeedLoader`, and could be part of `Root`, so we might call it as:
```swift
	return completion(.success(root.toFeedImages()))
```